apps/platform-app/src/app/[locale]/(profile)/_actions/mutations/publishProfile.ts

```diff
@@ -23,6 +24,8 @@ export const publishProfile = withSentry(async (profileId: string) => {
     state: PublishingState.PENDING,
   })
 
+  runEvaluateProfileAgent('', foundProfile.userId)
```

Comment:
**nit:** Nerdy is right - why empty string?

Reply:
we should pass just `profileId` basically

Reply:
yep, my bad
done